### DISCUSSION TITLE
一些外部内容

### DISCUSSION THREAD

**Discussion by Sylive147** [touhou-emperor/touhou-emperor.github.io#4] [Ideas]
可以提供某某维基和某某某维基的赵旭升词条备份作为补充阅读

**Comment by h476982844** [comment]
抱歉，这些东西一方面直接搜“车万皇帝”就能在第一页跳出来，另一方面我是拒绝放开盒信息的（就算他已经被开烂了）。做事没有底限那和姓赵的有什么区别。
